yaksh/templates/yaksh/course_modules.html

```diff
@@ -15,22 +15,17 @@
 {% block main %}
 
 <div class="container">
-<div class="row justify-content-md-center yakshwell ">
+<div class="row justify-content-md-center yakshwell">
     <div class="col-md-10 bg-light card">
-        <div class="row align-items-center">
+        <div class="row align-items-center" style="height: 100px;">
```

Comment:
Kindly avoid such inline CSS!

Reply:
@prathamesh920  Will it be okay to create a new class selector just for one tag in css file?

Reply:
Firstly, try to achieve it by proper use of classes provided by the bootstrap CSS.